common/src/main/java/com/wynntils/models/character/CharacterSelectionModel.java

```diff
@@ -1,165 +1,59 @@
 /*
- * Copyright © Wynntils 2022-2024.
+ * Copyright © Wynntils 2024.
```

Comment:
How does this happen, lol?

Reply:
Ah that's because I deleted the Model since it was only used for the screen, then added it back for implementing the 1-9 key press to select character